contracts/IERC20NonStandard.sol

```diff
@@ -7,9 +7,37 @@ pragma solidity 0.8.15;
  *  See https://medium.com/coinmonks/missing-return-value-bug-at-least-130-tokens-affected-d67bf08521ca
  */
 interface IERC20NonStandard {
```

Comment:
Looks like this is not used anymore. I did leave a comment suggesting we may need this again, but if not, we should remove this file.

Reply:
I think bulker seems still referencing to this file, right 🤔 ?

Reply:
Ah you're right, it is